packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -25,6 +30,10 @@ import {
 } from "@fluid-internal/test-version-utils";
 import { v4 as uuid } from "uuid";
 import { ConfigTypes, IConfigProviderBase } from "@fluidframework/telemetry-utils";
+// eslint-disable-next-line import/no-internal-modules
+import { PendingLocalState } from "@fluidframework/container-runtime/dist/test";
+// eslint-disable-next-line import/no-internal-modules
+import { IPendingBlobs } from "@fluidframework/container-runtime/dist/blobManager.js";
```

Comment:
add this to the above test index, don't directly import from js files

Reply:
Maybe I'm still doing something wrong. After moving IPendingBlobs to test index, (s as PendingLocalState).pendingAttachmentBlobs is still any

Reply:
changing to src fixed it